gossip/src/crds.rs

```diff
@@ -199,17 +200,22 @@ fn overrides(value: &CrdsValue, other: &VersionedCrdsValue) -> bool {
     // Contact-infos and node instances are special cased so that if there are
     // two running instances of the same node, the more recent start is
     // propagated through gossip regardless of wallclocks.
-    if let CrdsData::NodeInstance(value) = value.data() {
-        if let Some(out) = value.overrides(&other.value) {
-            return out;
+    match value.data() {
```

Comment:
should we just pass `data: &CrdsData` into this function instead?

Reply:
I think if we pass `CrdsData`, it kind of obfuscates `{ContactInfo,NodeInstance}::overrides` api.
Might be more self-explanatory if the api uses a more specific type;